README.md

```diff
@@ -31,57 +31,137 @@ An [Ansible AWX](https://github.com/ansible/awx) operator for Kubernetes built w
+serviceaccount/awx-operator created
+deployment.apps/awx-operator created
 ```
 
-Then create a file named `my-awx.yml` with the following contents:
+Wait a few minutes and you should have the `awx-operator` running.
+
+```bash
+$ kubectl get pods
+NAME                            READY   STATUS    RESTARTS   AGE
+awx-operator-7dbf9db9d7-z9hqx   1/1     Running   0          50s
+```
+
+Then create a file named `awx-demo.yml` with the suggested content. The `metadata.name` you provide, will be the name of the resulting AWX deployment.  If you deploy more than one AWX instance to the same namespace, be sure to use unique names.
 
 ```yaml
 ---
 apiVersion: awx.ansible.com/v1beta1
 kind: AWX
 metadata:
-  name: awx
+  name: awx-demo
+spec:
+  service_type: nodeport
+  ingress_type: none
+  hostname: awx-demo.example.com
 ```
 
-> The metadata.name you provide, will be the name of the resulting AWX deployment.  If you deploy more than one to the same namespace, be sure to use unique names.
-
 Finally, use `kubectl` to create the awx instance in your cluster:
 
 ```bash
-#> kubectl apply -f my-awx.yml
+$ kubectl apply -f awx-demo.yml
+awx.awx.ansible.com/awx-demo created
 ```
 
 After a few minutes, the new AWX instance will be deployed. One can look at the operator pod logs in order to know where the installation process is at. This can be done by running the following command: `kubectl logs -f deployments/awx-operator`.
 
-Once deployed, the AWX instance will be accessible at `http://awx.mycompany.com/` (assuming your cluster has an Ingress controller configured).
+```bash
+$ kubectl get pods -l "app.kubernetes.io/managed-by=awx-operator"
+NAME                        READY   STATUS    RESTARTS   AGE
+awx-demo-77d96f88d5-pnhr8   4/4     Running   0          3m24s
+awx-demo-postgres-0         1/1     Running   0          3m34s
+
+$ kubectl get svc -l "app.kubernetes.io/managed-by=awx-operator"
+NAME                TYPE        CLUSTER-IP     EXTERNAL-IP   PORT(S)        AGE
+awx-demo-postgres   ClusterIP   None           <none>        5432/TCP       4m4s
+awx-demo-service    NodePort    10.109.40.38   <none>        80:31006/TCP   3m56s
+```
+
+Once deployed, the AWX instance will be accessible by the command `minikube service awx-demo-service --url`.
 
 By default, the admin user is `admin` and the password is available in the `<resourcename>-admin-password` secret. To retrieve the admin password, run `kubectl get secret <resourcename>-admin-password -o jsonpath="{.data.password}" | base64 --decode`
 
+You just completed the most basic install of an AWX instance via this operator. Congratulations!!!!
+
+For an example using the Nginx Controller in Minukube, don't miss our [demo video](https://asciinema.org/a/416121).
 
-You just completed the most basic install of an AWX instance via this operator. Congratulations !
+[![asciicast](https://raw.githubusercontent.com/ansible/awx-operator/devel/docs/awx-demo.svg)](https://asciinema.org/a/416121)
```

Comment:
This doesn't seem to render correctly in the GitHub UI: https://github.com/ansible/awx-operator/blob/7f39d82c78ffd02376460272db6424cea1a21aef/README.md#network-and-tls-configuration

<img width="687" alt="image" src="https://user-images.githubusercontent.com/773186/119586869-82faff00-bd9b-11eb-96f7-d8e445e537dd.png">

Reply:
@shanemcd I guess that is because the `awx-demo.svg` is not in the  `ansible/awx-operator/devel` yet, right?  IF you point straight to the `svg` at https://raw.githubusercontent.com/tchellomello/awx-operator/ingress_minikube/docs/awx-demo.svg it works.